src/Plane/PlaneFileGlue.cpp

```diff
@@ -208,3 +210,21 @@ PlaneGlue::WriteFile(const Plane &plane, Path path)
   buffered.Flush();
   file.Commit();
 }
+
+StaticString<42>
+PlaneGlue::GetFileName(const Plane &plane)
+{
+  StaticString<42> filename(plane.registration);
+  filename += _T(".xcp");
+  return(filename);
+}
+
+// Simplified interface to write plane file including file extension handling
```

Comment:
API documentation should be in doxygen-style comments in the header

Reply:
to be done in PR for plane masses